exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/metric_exporter/__init__.py

```diff
@@ -79,6 +82,8 @@ def __init__(
             }
         )
 
+        self.max_export_batch_size: Optional[int] = max_export_batch_size
```

Comment:
```suggestion
        self._max_export_batch_size: Optional[int] = max_export_batch_size
```

Reply:
We try to keep the public attributes to a minimum. If you still consider `max_export_batch_size` should be public, please make it a property,